### DISCUSSION TITLE
Translate GUI to Dutch language

### DISCUSSION THREAD

**Discussion by KoalafiedDev** [dnSpyEx/dnSpy#213] [Ideas]
I don't know if there's any demand for this, but I'm able and willing to translate the GUI to Dutch.

**Comment by GazziFX** [comment]


**Reply by KoalafiedDev** [reply]
That's German 😉. But I already found the [link](https://github.com/dnSpyEx/dnSpy#:~:text=Click%20here%20if%20you%20want%20to%20help%20with%20translating%20dnSpy%20to%20your%20native%20language.) to contribute to the translations!

**Comment by KoalafiedDev** [comment]
I'm going to use the link: https://crowdin.com/project/dnspy/nl to contribute to the translations.
